wg-serving/charter.md

```diff
@@ -0,0 +1,87 @@
+# WG Serving Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [wg-governance].
+
+[Kubernetes Charter README]: /committee-steering/governance/README.md
+
+## Scope
+
+Discuss and enhance the support for AI/ML inference workloads in Kubernetes.
+
+
+### In scope
+
+- Gather requirements for serving workloads (inference primarily, but benefiting
+  other non-batch use cases where possible) that have broad community alignment
+  from practitioners, distros, and vendors. Provide concrete input to other SIGs
+  and WGs around needs for identified requirements. Do it in partnership
+  with existing ecosystem projects like kServe, Seldon, Kaito, and
+  others to identify, extract, or implement common shared problems (like Kueue
+  abstracted deferred scheduling for multiple batch frameworks).
+- Specific areas of improvement include:
+  - Directly improve key kubernetes workload controllers when used with
+    accelerators and the most common inference serving frameworks and model
+    servers.
+  - Explore new projects that improve orchestration, scaling, and load balancing
+    of inference workloads and compose well with other workloads on Kubernetes
+  - Being able to run serving workloads safely while giving up
+    available slack capacity to batch frameworks
+
+### Out of scope
+
+- Batch inference, which should be covered by WG Batch.
```

Comment:
Training?

Reply:
this refers to batch jobs that will run a batch of inference queries nighty or some similar scenario. So basically something the has characteristics similar to training. Does it make sense or some better language here will help?

Reply:
I guess it might still be worth adding batch training to the list of "out of scope", for additional clarity.

Reply:
Agreed, let's say "training and batch inference are under WG-Batch".

Continuous training may eventually be relevant, but we don't have to call it out here.